ci-operator/step-registry/aws/provision/tags-for-byo-vpc/pre/aws-provision-tags-for-byo-vpc-pre-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: aws-provision-tags-for-byo-vpc-pre
```

Comment:
As this step is only for pre-flight check, I suggest to add a `ocm`/`managed-cluster` or something like that to the step name, e.g. `aws-provision-tags-for-byo-vpc-for-mc` or `aws-provision-tags-for-byo-vpc-for-mc-precheck`

Reply:
The step is not only for ocm, but now it is only used by ocm. Need us to limit it only for ocm?